src/api/schemas/extendedQueryNode.graphql

```diff
@@ -9,137 +9,130 @@ type Member {
   handle: String!
 }
 
-type Channel {
-  id: ID!
+enum AssetAvailability {
+  ACCEPTED
+  PENDING
+  INVALID
+}
 
-  createdAt: Date!
+enum LiaisonJudgement {
+  PENDING
+  ACCEPTED
+  REJECTED
+}
 
-  handle: String!
+type DataObject {
+  id: ID!
+  createdAt: DateTime!
+  size: Int!
+  # id of storage provider
+  liaisonId: Int!
+  # Storage provider as liaison judgment
+  liaisonJudgement: LiaisonJudgement!
+  # IPFS content id
+  ipfsContentId: String!
+  # Joystream runtime content
+  joystreamContentId: String!
+}
 
-  description: String
+type Block {
+  id: ID!
+  block: Int!
+}
 
+type Channel {
+  id: ID!
+  createdAt: DateTime!
+  title: String!
+  description: String
+  coverPhotoDataObject: DataObject
   coverPhotoUrl: String
-
+  coverPhotoAvailability: AssetAvailability!
+  avatarPhotoDataObject: DataObject
   avatarPhotoUrl: String
-
+  avatarPhotoAvailability: AssetAvailability!
   owner: Member!
-
-  isPublic: Boolean!
-
-  isCurated: Boolean!
-
+  isPublic: Boolean
+  isCensored: Boolean
   language: Language
-
   videos: [Video!]!
 
   # extended from Orion
   follows: Int
 }
 
-type Category {
+type VideoCategory {
   id: ID!
-
-  name: String!
-
-  videos: [Video!]
-}
-
-type JoystreamMediaLocation {
-  dataObjectId: String!
-}
-
-type HttpMediaLocation {
-  url: String!
-}
-
-# In the future we can add IPFS, Dat, etc.
-union MediaLocation = JoystreamMediaLocation | HttpMediaLocation
-
-type KnownLicense {
-  code: String!
   name: String
-  description: String
-  url: String
-}
-
-type UserDefinedLicense {
-  content: String!
+  videos: [Video!]
 }
 
-union License = UserDefinedLicense | KnownLicense
-
-type LicenseEntity {
+type License {
   id: ID!
-  type: License!
+  code: Int
+  url: String
   attribution: String
   videoLicense: [Video!]
+  customText: String
 }
-type VideoMedia {
+
+type VideoMediaMetadata {
   id: ID!
 
   # Resolution width
-  pixelWidth: Int!
+  pixelWidth: Int
 
   # Resolution height
-  pixelHeight: Int!
+  pixelHeight: Int
   # Size in bytes
   size: Float
-
-  # where to find
-  location: MediaLocation!
 }
 
 type Video {
   id: ID!
-
   channel: Channel!
-
-  category: Category!
-
+  createdAt: DateTime!
+  createdById: String!
+  updatedAt: DateTime
+  updatedById: String
```

Comment:
Let's remove both `ById` fields. It's not really clear what they represent so it's best to avoid confusion